gitops-cluster.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
```

Comment:
This file is already checked in, please make sure this doesn't override the existing file. Seems like a rebase issue.